arm/Microsoft.ServiceBus/namespaces/readme.md

```diff
@@ -48,6 +48,42 @@ This module deploys Service Bus Namespace resource.
 | `workspaceId` | string |  |  | Optional. Resource ID of log analytics. |
 | `zoneRedundant` | bool |  |  | Optional. Enabling this property creates a Premium Service Bus Namespace in regions supported availability zones. |
 
+### Parameter Usage: `privateEndpoints`
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done